frontend/src/core/comments/components/AddComment.tsx

```diff
@@ -1,30 +1,55 @@
-/* @flow */
-
 import * as React from 'react';
 import ReactDOM from 'react-dom';
 import { Localized } from '@fluent/react';
-import { Editor, Transforms, Range, createEditor, Text, Node } from 'slate';
+import {
+    BaseEditor,
+    Editor,
+    Element as SlateElement,
+    Transforms,
+    Range,
+    createEditor,
+    Text,
+    Node,
+} from 'slate';
+import type { Descendant } from 'slate';
```

Comment:
This can be included in the above 'slate' imports unless we want to specifically call out that it is a `type` import

Reply:
Yeah, pointing out that this is just a type import was intentional. I cross-checked the other imports that they're actually calling into the API layer there.